includes/apple-exporter/builders/class-components.php

```diff
@@ -163,11 +168,37 @@ private function meta_components() {
 		// Get the component order
 		$meta_component_order = $this->get_setting( 'meta_component_order' );
 		if ( ! empty( $meta_component_order ) && is_array( $meta_component_order ) ) {
-			foreach ( $meta_component_order as $component ) {
-				$method = 'content_' . $component;
-				if ( method_exists( $this, $method ) && $this->$method() ) {
-					$components[] = $this->get_component_from_shortname( $component, $this->$method() )->to_array();
+			if ( 'cover' !== $meta_component_order[0] ) {
+				foreach ( $meta_component_order as $component ) {
+					$method = 'content_' . $component;
+					if ( method_exists( $this, $method ) && $this->$method() ) {
+						$components[] = $this->get_component_from_shortname( $component, $this->$method() )->to_array();
+					}
+				}
+			} else {
+				$nesting = array();
+				foreach ( $meta_component_order as $component ) {
+					$method = 'content_' . $component;
+					if ( method_exists( $this, $method ) && $this->$method() ) {
+						if ( 'cover' === $component ) {
+							$components[] = $this->get_component_from_shortname( $component, $this->$method() )->to_array();
+						} else {
+							$nesting[] = $this->get_component_from_shortname( $component, $this->$method() )->to_array();
+						}
```

Comment:
it's pretty clear *what* is happening here, but it might be worth a little comment to explain *why* -- when cover is the first component, the rest get grouped into a container, but if cover were a third component no special handling?